workflows/cloud-client/src/test/java/com/example/workflows/WorkflowsQuickstartTest.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2021 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.workflows;
+
+import static com.example.workflows.WorkflowsQuickstart.workflowsQuickstart;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertNotNull;
+
+import com.google.cloud.workflows.v1.Workflow;
+import com.google.cloud.workflows.v1.WorkflowName;
+import com.google.cloud.workflows.v1.WorkflowsClient;
+import java.io.IOException;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class WorkflowsQuickstartTest {
+
+  private static String projectId;
+  private static final String LOCATION_ID = "us-central1";
+  private static final String WORKFLOW_ID = "myFirstWorkflow";
+
+  // Workflow source copied from:
+  // https://github.com/GoogleCloudPlatform/workflows-samples/blob/main/src/myFirstWorkflow.workflows.yaml
+  private static final String WORKFLOW_SOURCE = "# Copyright 2020 Google LLC\n#\n# Licensed under the Apache License, Version 2.0 (the \"License\");\n# you may not use this file except in compliance with the License.\n# You may obtain a copy of the License at\n#\n# http://www.apache.org/licenses/LICENSE-2.0\n#\n# Unless required by applicable law or agreed to in writing, software\n# distributed under the License is distributed on an \"AS IS\" BASIS,\n# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n# See the License for the specific language governing permissions and\n# limitations under the License.\n\n# [START workflows_myfirstworkflow]\n- getCurrentTime:\n call: http.get\n args:\n url: https://us-central1-workflowsample.cloudfunctions.net/datetime\n result: currentTime\n- readWikipedia:\n call: http.get\n args:\n url: https://en.wikipedia.org/w/api.php\n query:\n action: opensearch\n search: ${currentTime.body.dayOfTheWeek}\n result: wikiResult\n- returnResult:\n return: ${wikiResult.body[1]}\n# [END workflows_myfirstworkflow]\n";
```

Comment:
Is there any way to make this easy to read? I understand this is not the focus of this sample, but it would be nicer to read from a YAML

Reply:
It's a good question. I'd rather be able to quickly update the test with a copy/paste than format this. If we run on Java 15+, we could use text blocks.

Reply:
We can leave as is, but is there another sample we can link to that shows deploying of the YAML?